pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -3519,7 +3519,7 @@ in
   vim2nix = buildVimPlugin {
     pname = "vim2nix";
     version = "1.0";
-    src = ./vim2nix;
+    src = ./non-generated/vim2nix;
     dependencies = [ self.vim-addon-manager ];
   };
```

Comment:
Should package definitions that aren't "overriding" anything be move out of `overrides.nix` and into another file?

I wonder if we could take a semi-automated approach, similar in principal to other by-name trees, where each directory in `non-generated` gets imported into `vimPlugins` via `callPackage`, using the directory name as the plugin attr name.

Reply:
Yeah, I think we should basically have a non-generated.nix or something that houses them instead of being inside overrides.

Reply:
Started implementing it. Didn't realize how many we had in there that were new plugin definitions in overrides.nix